tests/tests_lite/conftest.py

```diff
@@ -80,13 +76,6 @@ def teardown_process_group():
         torch.distributed.destroy_process_group()
 
 
-@pytest.fixture(scope="function", autouse=True)
-def reset_deterministic_algorithm():
-    """Ensures that torch determinism settings are reset before the next test runs."""
-    yield
-    torch.use_deterministic_algorithms(False)
```

Comment:
@carmocca This is a good fixture to have. There will be tests in the launchers that set and reset this.

Reply:
I thought this was for `Trainer(deterministic=True)` tests. Do any Lite tests set this?

Reply:
There will be one case yes, here at the bottom: https://github.com/Lightning-AI/lightning/pull/14555/files#diff-75699c31ae673f40bd55d15a6179fc51a8bb77890a6f0e780b9f3b1ce877fca8

Not a big deal, I can also add it back in that PR

Reply:
Yeah, and also just for that test instead of a fixture. They must incur a fixed overhead per test if autoused